DS13/code/modules/borg/conversion.dm

```diff
@@ -0,0 +1,366 @@
+				target.Jitter(3)
+				playsound(target.loc, 'DS13/sound/effects/borg/grab.ogg', 100,1)
+				assimilate(user, target)
+		if(mode == "build")
+			if(istype(I, /turf/open))
+				var/turf/open/T = I
+				build_on_turf(T, user)
+				return
+	else
+		. = ..()
+
+/obj/item/borg_tool/proc/tear_airlock(obj/machinery/door/airlock/A, mob/user)
+	removing_airlock = TRUE
+	to_chat(user,"<span class='notice'>You start tearing apart the airlock...\
+		</span>")
+	playsound(src.loc, 'DS13/sound/effects/borg/machines/borgforcedoor.ogg', 100, 4)
+	A.audible_message("<span class='italics'>You hear a loud metallic \
+		grinding sound.</span>")
+	if(do_after(user, delay=80, needhand=FALSE, target=A, progress=TRUE))
+		A.audible_message("<span class='danger'>[A] is ripped \
+			apart by [user]!</span>")
+		qdel(A)
+	removing_airlock = FALSE
+
+
+
+/obj/structure/chair/borg/conversion
+	name = "assimilation bench"
+	desc = "Submit yourself to the collective"
+	icon_state = "borg_off"
+	icon = 'DS13/icons/borg/borg.dmi'
+	anchored = 1
+	can_buckle = 1
+	can_be_unanchored = 0
+	max_buckled_mobs = 1
+	buildstacktype = null
+	item_chair = null // if null it can't be picked up
+	var/restrained = 0 //can they unbuckle easily?
+
+
+/obj/structure/chair/borg/conversion/user_buckle_mob(mob/living/carbon/human/M, mob/user)
+	if(!isBorgDrone(M))
+		to_chat(M, "Assimilate them first")
+		return
+	M.unequip_everything()
+	icon_state = "borg_off"
+	visible_message("parts of [src] start to shift and move")
+	playsound(loc, 'DS13/sound/effects/borg/machines/convert_table.ogg', 50, 1, -1)
+	var/image/armoverlay = image('DS13/icons/borg/borg.dmi')
+	armoverlay.icon_state = "borg_arms"
+	armoverlay.layer = ABOVE_MOB_LAYER
+	overlays += armoverlay
+	var/image/armoroverlay = image('DS13/icons/borg/borg.dmi')
+	armoroverlay.icon_state = "borgarmour"
+	armoroverlay.layer = ABOVE_MOB_LAYER
+	overlays += armoroverlay
+	sleep(35)
+	playsound(loc, 'DS13/sound/effects/borg/machines/convert_table2.ogg', 50, 1, -1)
+	icon_state = "borg_off"
+	M.equipOutfit(/datum/outfit/borg, visualsOnly = FALSE) //Outfit handles name etc.
+	cut_overlays()
+	overlays -= armoverlay
+	overlays -= armoroverlay
+	qdel(armoroverlay)
+	qdel(armoverlay)
+	return TRUE
+
+
+/obj/structure/chair/borg/charging
+	name = "recharging alcove"
+	desc = "We must recharge to regain"
+	icon_state = "borgcharger"
+	icon = 'DS13/icons/borg/borg.dmi'
+	anchored = TRUE
+	can_buckle = TRUE
+	can_be_unanchored = FALSE
+	max_buckled_mobs = 1
+	resistance_flags = FIRE_PROOF
+	buildstacktype = null
+	item_chair = null // if null it can't be picked up
+	var/cooldown2 = 110 //music loop cooldowns
+	var/saved_time2 = 0
+	var/sound = 'DS13/sound/effects/borg/machines/alcove.ogg'
+	var/mob/living/carbon/human/user
+	var/charge_time = 0
+	var/charge_amount = 10 //10 per tick, with a max charge storage of 1000
+
+/obj/structure/chair/borg/charging/New()
+	. = ..()
+	START_PROCESSING(SSobj, src)
```

Comment:
On buckle and remove from buckle

Reply:
Unable to do this as it doesn't work (process ceases to function)